src/Features/CSharp/Portable/AddImport/CSharpAddImportFeatureService.cs

```diff
@@ -132,6 +132,10 @@ protected override bool CanAddImport(SyntaxNode node, CancellationToken cancella
         protected override bool CanAddImportForDeconstruct(string diagnosticId, SyntaxNode node)
             => diagnosticId == CS8129;
 
+        protected override bool CanAddImportForGetAwaiter(string diagnosticId, SyntaxNode node) =>
+            diagnosticId == CS1061 &&
+            node.AncestorsAndSelf().Any(n => n is AwaitExpressionSyntax);
```

Comment:
📝 See other discussion, but this subexpression might be better as `node.FirstAncestorOrSelf<AwaitExpressionSyntax>() != null`